sample_apps/run_example.sh

```diff
@@ -94,11 +108,22 @@ function usage() {
    echo "  To run an individual step of the ${SampleAppName}         : ./${Me} ${SampleAppName} <step name>"
 }
 
+# ---- Open-Enclave app-specific help/usage output
+function usage_OE() {
+   echo
+   echo "For simple_app_under_oe, you can alternatively use this script "
+   echo "to generate the policy by editing the measurement in the policy JSON file:"
+   echo "  To setup the example program        : ./${Me} simple_app_under_oe setup_with_generate_policy"
+   echo "  To run and test the example program : ./${Me} run_test"
+}
+
 # ###########################################################################
 # Dump the list of functions, one for each step, that user can invoke.
 # Ref: https://www.freecodecamp.org/news/bash-array-how-to-declare-an-array-of-strings-in-a-bash-script/
 # ###########################################################################
 Steps=( "rm_non_git_files"
+
```

Comment:
Each application now has its own set of "Steps"; i.e. a list of functions implementing one part of the procedure.